fs/dafsclient/rmtclient.cpp

```diff
@@ -1229,10 +1238,12 @@ ISocket *connectDafs(SocketEndpoint &ep, unsigned timeoutms, const IPropertyTree
         {
             conAttempts--;
             bool connected = false;
+            CCycleTimer timer;
             try
             {
                 socket.setown(ISocket::connect_timeout(ep, newtimeout));
                 connected = true;
+                remainingMs = timer.remainingMs(newtimeout);
```

Comment:
not sure this is right. Let's say connectDafs (.., timeoutms=60000, ..)
newtimeout will be be set to 5000.
connect_timeout taks 4900 ms, remainingMs = 100
checkSocketSecure called with timeout of 100ms (but the timeout was supposed to be 60s overall).

maybe should be:
```
remainingMs = timer.remainingMs(timeoutms);
```
but CCycleTimer should be outside of attempts loop too I think.